common/group.cpp

```diff
@@ -84,6 +84,8 @@ std::ostream& operator<<(std::ostream& ss, fty::Group::Fields value)
                 return "subtype";
             case fty::Group::Fields::Type:
                 return "type";
+            case fty::Group::Fields::InternalName:
+                return "internal-name";
```

Comment:
The field will contain an internal name of an asset, but the field itself is named "asset"